MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+# setuptool_scm takes care of including everything committed to git, while
```

Comment:
Minimal MANIFEST.in good, yes.

How have you checked that you get the files you expect? When tuning MANIFEST.in/setup.py/setup.cfg, I usually do a fake build and diff the dir-listing in both the wheel (zip) and tarball.

Would it be worth it to have a build-test?

Reply:
I've only made rudimentary checks. In working more with this last week, including sending some upstream PRs to the libsass-python project, it seems the MANIFEST.in may be wholly unnecessary. 

We'll see as we attempt to split up this into multiple PRs.